packages/fuels-core/src/types/transaction_builders.rs

```diff
@@ -337,48 +328,64 @@ impl ScriptTransactionBuilder {
     fn create_dry_run_witnesses(&self, num_witnesses: u8) -> Vec<Witness> {
```

Comment:
A comment here would be useful so we don't forget why we do this

Reply:
There is a comment just above the `fn`